ex02/ex02.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+for path in $(echo $1 | tr " " "\n")
+do
+	UNEXISTING=true
```

Comment:
we usually set tab size as 4 spaces
```suggestion
    UNEXISTING=true
```